src/streaming/MediaPlayer.js

```diff
@@ -1043,6 +1043,25 @@ MediaPlayer = function (context) {
         },
 
         /**
+         * Use this method to attach an HTML5 div for dash.js to render rich TTML subtitles.
+         *
+         * @param div An unstyled div placed after the video element. It will be styled to match the video size and overlay z-order.
+         * @memberof MediaPlayer#
+         */
+        attachTTMLRenderingDiv: function (div) {
+            if (!initialized) {
+                throw "MediaPlayer not initialized!";
+            }
+
+
+            videoModel = null;
```

Comment:
I do not think we should set this to null.  VideoModel already created in the media player.  What is the reason?